manifests/cluster/patch/images.yaml

```diff
@@ -26,7 +26,9 @@ spec:
             value: quay.io/goharbor/notary-signer-photon:v2.1.2
           - name: TRIVY_DOCKER_IMAGE_2_1_2
```

Comment:
This is by design. The image from the configuration has high priority than the default image, it will break the upgrading when the image from the configuration is not related to the harbor version.